jihyeon/app.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const cors = require('cors');
+const morgan = require ("morgan");
+const http = require ("http");
+require("dotenv").config();
+
+const { DataSource } = require('typeorm');
+
+const dataSource = new DataSource({
+    type: process.env.DB_CONNECTION,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    username: process.env.DB_USER,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DATABASE
+})
+
+dataSource.initialize()
+    .then(() => {
+       console.log("Data Source has been initialized!")
+   })
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('dev'));
+
+app.get('/ping', function (req, res, next){
+    res.json({message: 'pong'})
+});
+
+const server = http.createServer(app);
+const PORT = process.env.PORT;
+
+const start = async () => {
+    server.listen(PORT, () => console.log(`server is listening on ${PORT}`));
```

Comment:
```suggestion
    app.listen(PORT, () => console.log(`server is listening on ${PORT}`));
```
http를 쓰지 않기 때문에, express로 만들어준 app이라는 변수를 활용하여 listen하면 됩니다.